client/src/routes/Session/ChangingRoom.tsx

```diff
@@ -138,20 +138,44 @@ const ChangingRoom = () => {
   const hasVideo = Boolean(me?.videoTrack);
 
   useEffect(() => {
-    if (session?.id) {
+    // If switching between sessions, the session will first be the old one
```

Comment:
Does switching between sessions means leaving a session and entering another one through the UI?
I thought we cleared the session state rather than keeping the old session on 🤯

Reply:
Found this when going back from the `ChangingRoom` for different sessions, so it has probably not been happening too much but still, better to be a 100%

Reply:
So not when leaving the regular way